app/scripts/directives/updateTimestamp.js

```diff
@@ -0,0 +1,224 @@
+'use strict';
+
+/**
+ * @ngdoc directive
+ * @name oncokb.directive:updateTimestamp
+ * @author Jing Su created on 2019/06/26
+ * @description
+ * # refresh date
+ */
+angular.module('oncokbApp')
+    .directive('updateTimestamp', function(_, $timeout, mainUtils, DatabaseConnector, FirebaseModel, $rootScope, firebaseConnector) {
+        return {
+            templateUrl: 'views/updateTimestamp.html',
+            restrict: 'E',
+            scope: {
+                type: '=',
+                hugoSymbol: '=',
+                key: '=',
+                obj: '=',
+                disabled: '=',
+                updateTime: '=',
+                updatedBy: '=',
+                path: '='
+            },
+            replace: false,
+            controller: function($scope) {
+                $scope.clicked = false;
```

Comment:
The name does not explain the meaning of the actual use-case

Reply:
And I do think the status should be wrapped in a more meaningful function, something like `afterUpdateValidateTime` then it should accept a status and message.